tests/queries/0_stateless/02417_opentelemetry_insert_on_distributed_table.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, distributed
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+function insert()
+{
+    echo "INSERT INTO ${CLICKHOUSE_DATABASE}.dist_opentelemetry SETTINGS insert_distributed_sync=$2 VALUES(1),(2)" |
+        ${CLICKHOUSE_CURL} \
+            -X POST \
+            -H "traceparent: 00-$1-5150000000000515-01" \
+            -H "tracestate: some custom state" \
+            "${CLICKHOUSE_URL}" \
+            --data @-
+}
+
+function check_span()
+{
+${CLICKHOUSE_CLIENT} -nq "
+    SYSTEM FLUSH LOGS;
+
+    SELECT count() FROM system.opentelemetry_span_log
+    WHERE lower(hex(trace_id))                = '${1}'
+    AND   operation_name                      like '${2}'
+    AND   attribute['clickhouse.shard_num']   = '${3}'
+    AND   attribute['clickhouse.cluster']     = 'test_cluster_two_shards'
+    AND   attribute['clickhouse.distributed'] = '${CLICKHOUSE_DATABASE}.dist_opentelemetry'
+    AND   attribute['clickhouse.remote']      = '${CLICKHOUSE_DATABASE}.local_opentelemetry'
+    AND   attribute['clickhouse.rows']        = '1'
+    AND   attribute['clickhouse.bytes']       = '8'
+    ;"
+}
+
+
+#
+# Prepare tables for tests
+#
+${CLICKHOUSE_CLIENT} -nq "
+DROP TABLE IF EXISTS ${CLICKHOUSE_DATABASE}.dist_opentelemetry;
+DROP TABLE IF EXISTS ${CLICKHOUSE_DATABASE}.local_opentelemetry;
```

Comment:
You can omit `CLICKHOUSE_DATABASE` in `CREATE`/`DROP`, though it is minor.

Reply:
Actually, when I remove the database from the the CREATE/DROP, the tables are not correctly created under the `${CLICKHOUSE_DATABASE}` database but under `default` database. We should add `--database` to the command so that correct database will be applied, so I didn't make change to these queries.

Reply:
This is done automatically by `shell_config.sh` - https://github.com/ClickHouse/ClickHouse/blob/9c5f9f1815a9f9a9dfded85b260b2b201376498d/tests/queries/shell_config.sh#L21

So this should work, but like I said this is minor, and if you don't have any other changes you can ignore this. But if you have something else, then please apply this suggestion.